data/templates/default.latex

```diff
@@ -1,15 +1,66 @@
 \documentclass[10pt,a4paper,onecolumn]{article}
 \usepackage{marginnote}
 \usepackage{graphicx}
-\usepackage{xcolor}
+\usepackage[rgb]{xcolor}
 \usepackage{authblk,etoolbox}
 \usepackage{titlesec}
 \usepackage{calc}
 \usepackage{tikz}
-\usepackage{hyperref}
-\hypersetup{colorlinks,breaklinks=true,
-            urlcolor=[rgb]{0.0, 0.5, 1.0},
-            linkcolor=[rgb]{0.0, 0.5, 1.0}}
+\usepackage[pdfa]{hyperref}
+\usepackage{hyperxmp}
+\usepackage{embedfile}
+\hypersetup{%
+    unicode=true,
+    pdfapart=3,
+    pdfaconformance=B,
+    $if(title-meta)$
+    pdftitle={$title-meta$},
+    $endif$
+    $if(author-meta)$
+    pdfauthor={$author-meta$},
+    $endif$
+    $if(keywords)$
+    pdfkeywords={$for(keywords)$$keywords$$sep$; $endfor$},
+    $endif$
+    pdfpublication={$journal_name$},
+    pdfpublisher={Open Journals},
+    pdfissn={$journal_issn$},
+    pdfpubtype={journal},
+    pdfvolumenum={$volume$},
+    pdfissuenum={$issue$},
+    $if(formatted_doi)$
+    pdfdoi={$formatted_doi$},
+    $endif$
+    $if(author-meta)$
+    pdfcopyright={Copyright (c) $year$, $author-meta$},
+    $endif$
```

Comment:
I don't know which of these variables need to be treated as optional and which can just be used.